src/fosslight_source/_parsing_scancode_file_item.py

```
@@ -207,6 +207,19 @@ def split_spdx_expression(spdx_string: str) -> list:
     return license
 
 
+def get_license_expression_spdx(license_expression: str) -> str:
+    if not license_expression or not license_expression.strip():
+        return ""
+    try:
+        from licensedcode.cache import build_spdx_license_expression
+        result = build_spdx_license_expression(license_expression.strip())
+        if regex.match(result):
+            return ""
+        return result
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)